internal/cli/deployments/options/deployment_opts_remove.go

```diff
@@ -31,10 +31,6 @@ func (opts *DeploymentOpts) RemoveLocal(ctx context.Context) error {
 		return errRemove
 	}
 
-	if _, errRemove := opts.PodmanClient.RemoveNetworks(ctx, opts.LocalNetworkName()); errRemove != nil {
```

Comment:
as we clean up things, I left a comment in the [scope](https://docs.google.com/document/d/1c5xU5trWBMf7H0IOcPMi_Fj2gyl_JnJpe2QAOzrcIOY/edit?disco=AAABLbO9g8s).

- will these changes be released in to a major version?
- as customers move to a new version of the CLI, how do we make sure the leftover instances from podman are removed? 
- is it OK to add this code here and perform a release this week?